src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
I also think that `auto + series`  would be the best default when we will allow non linear ticks **auto**matic generation: for example, let's say weekend are excluded, generated ticks would be: ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Monday", "Tuesday", ...] and we will want the same distance between each ticks (`ticks.mode: 'series'`). If you set `ticks.mode: 'linear'`, the distance between each "Friday" and "Monday" will be twice the distance between other days.